nmigen/blink.py

```diff
@@ -0,0 +1,31 @@
+# Blink the three LEDs in a counting pattern.
+
+from nmigen import *
+from nmigen_boards.orangecrab_r0_2 import *
+
+
+class Blink(Elaboratable):
+    def __init__(self):
+        self.count = Signal(32, reset=0)
+
+    def elaborate(self, platform):
+        m = Module()
+
+        m.d.sync += self.count.eq(self.count + 1)
+        rgb = platform.request('rgb_led', 0)
+        red_led = rgb.r
+        green_led = rgb.g
+        blue_led = rgb.b
+
+        m.d.comb += [
+            red_led.o.eq(self.count[28]),
+            green_led.o.eq(self.count[27]),
+            blue_led.o.eq(self.count[26]),
+        ]
+
+        return m
+
+
+if __name__ == "__main__":
+    platform = OrangeCrabR0_2Platform()
+    platform.build(Blink())
```

Comment:
you can add do_program=True here to skip needing the dfu-util step.

Reply:
That's a good point, thanks. I've updated `blink.py` and `README.md`.